src/com/estafet/learning/US008/Order.java

```diff
@@ -0,0 +1,27 @@
+package com.estafet.learning.US008;
+
+public abstract class Order {
+    public static final String ORDER_NAME = "ORR";
+    public String OrderNumber;
+    public String ClientDetails;
+    public String ListWithArticles;
+    public String TotalAmount;
+    private static final int invoice = 1;
+    public static final String color = "green";
+
+    public Order(){
```

Comment:
it's default constructor - no need to define it like this unless you want to cusomize it

Reply:
it was overrided